browser/.beforeprettier

```diff
@@ -122,6 +122,7 @@
 /src/layer/marker/TextInput.js
 /src/layer/SplitPanesContext.ts
 /src/canvas/sections/AutoFillMarkerSection.ts
+/src/canvas/sections/TableFillMarkerSection.ts
```

Comment:
please do not add new files here,
it is blocking the code auto-formatter verification